### PR TITLE
docs: add uninstall instructions

### PR DESCRIPTION
**Pull request by jsgalarraga**
## Status

**READY**

## Description

#709

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: docs/docs/overview.md [hunk 1]
```diff
@@ -69,6 +69,22 @@ Create a production build which includes a `DockerFile` so that you can deploy a
 dart_frog build
 ```
 
+### Uninstalling 🗑️
```
Comment (alestiago) [inline docs/docs/overview.md:72]: This is currently the section "Quick Start 🚀",  and I think it doesn't make much sense to have it there. Maybe:

```suggestion
## Uninstalling 🗑️
```

File: docs/docs/overview.md [hunk 2]
```diff
@@ -69,6 +69,22 @@ Create a production build which includes a `DockerFile` so that you can deploy a
...
+
+To uninstall dart frog completely, the cli completion files have to be removed before uninstalling.
```
Comment (alestiago) [inline docs/docs/overview.md:74]:
```suggestion
To uninstall Dart Frog completely, the [CLI completion](https://github.com/VeryGoodOpenSource/cli_completion) files have to be removed before uninstalling.
```

File: docs/docs/overview.md [hunk 3]
```diff
@@ -69,6 +69,22 @@ Create a production build which includes a `DockerFile` so that you can deploy a
...
+
+```shell
+# 🗑️ Uninstall the dart_frog cli completion files
+dart_frog uninstall-completion-files
+```
+
+Now, dart frog can be uninstalled safely.
```
Comment (alestiago) [inline docs/docs/overview.md:81]:
```suggestion
Now, if you installed Dart Frog globally via Pub, Dart's package manager, you can uninstall Dart Frog using:
```

File: docs/docs/overview.md [hunk 4]
```diff
@@ -69,6 +69,22 @@ Create a production build which includes a `DockerFile` so that you can deploy a
 dart_frog build
 ```
 
+### Uninstalling 🗑️
+
+To uninstall dart frog completely, the cli completion files have to be removed before uninstalling.
+
+```shell
+# 🗑️ Uninstall the dart_frog cli completion files
```
Comment (alestiago) [inline docs/docs/overview.md:77]:
```suggestion
# 🗑️ Uninstall the Dart Frog CLI completion files
```

File: docs/docs/overview.md [hunk 5]
```diff
@@ -69,6 +69,22 @@ Create a production build which includes a `DockerFile` so that you can deploy a
...
+dart_frog uninstall-completion-files
+```
+
+Now, dart frog can be uninstalled safely.
+
+```shell
+# 🗑️ Uninstall the dart_frog cli from pub.dev
```
Comment (alestiago) [inline docs/docs/overview.md:84]:
```suggestion
# 🗑️ Uninstall the Dart Frog CLI from pub.dev
```

**Review by alestiago — DISMISSED** [review]

File: docs/docs/overview.md [hunk 6]
```diff
@@ -69,6 +69,22 @@ Create a production build which includes a `DockerFile` so that you can deploy a
 dart_frog build
 ```
 
+## Uninstalling 🗑️
+
+To uninstall Dart Frog completely, the [CLI completion](https://github.com/VeryGoodOpenSource/cli_completion) files have to be removed before uninstalling.
+
+```shell
+# 🗑️ Uninstall the Dart Frog CLI completion files
```
Comment (wolfenrain) [inline docs/docs/overview.md:77]:
```suggestion
# 🧹 Removing the Dart Frog CLI completion files
```

**Review by scarletteliza — APPROVED** [review]

**Review by renancaraujo — APPROVED** [review]

**Pull request merged**